### PR TITLE
Fix sync link resolution

### PR DESCRIPTION
**Pull request by axe312ger**
the sync API sends deleted entries without fields. This broke the link resolution. This small bug fix fixes the issue :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,6 +253,9 @@ Your Space ID.
 #### host (default: `'cdn.contentful.com'`)
 Set the host used to build the request URI's.
 
+#### basePath (default: ``)
```
Comment (Khaledgarbaya) [inline README.md]: why is this here ?

Comment (axe312ger) [inline README.md]: Because this also forces to upgrade to the latest sdk-core version which includes that feature

Comment (googol) [inline README.md]: It's missing from `index.d.ts`

Comment (axe312ger) [inline README.md]: @googol fixed 🙈  - we consider adding tests for this in the next version so it won't happen again :)
